src/Nethermind/Nethermind.Core/Specs/IReleaseSpec.cs

```diff
@@ -419,5 +419,10 @@ public interface IReleaseSpec : IEip1559Spec, IReceiptSpec
         bool IsAuthorizationListEnabled => IsEip7702Enabled;
 
         public bool RequestsEnabled => ConsolidationRequestsEnabled || WithdrawalRequestsEnabled || DepositsEnabled;
+
+        /// <summary>
+        /// Skip read in SSTORE for calculating gas cost as not charged
+        /// </summary>
+        bool IsSystemTransaction { get; }
```

Comment:
Looks weird, this is added to ReleaseSpec and the field seems to be referenced to Transaction object

Reply:
I agree this is a bit weird way of passing that information and is not universal